Templates/v2/Kotlin/PackageSeparation.md

```diff
@@ -0,0 +1,41 @@
+# SurfGen Kotlin package separation
```

Comment:
Did you add link on this file from README?

Reply:
Sure